### DISCUSSION TITLE
Manual input for draggable slider values

### DISCUSSION THREAD

**Discussion by Stoppedpuma** [ppy/osu#27702] [Ideas] [answered]
Might be a duplicate but I can't find anything while searching, originally brought up [here](https://github.com/ppy/osu/discussions/27526#discussioncomment-8874010).

This is especially useful on mobile and useful as well on desktop to get precise values. My idea of how to access this would be right clicking the draggable part on desktop and holding it on mobile to show a text input box.

Proof of concept design:

![manual_input](https://github.com/ppy/osu/assets/58333920/5b9858d7-f936-43c8-a696-14fb3fb62c25)

**Comment by sz-da** [comment, marked as answer]
probably duplicate of https://github.com/ppy/osu/discussions/24900
